crates/release_plz_core/src/project.rs

```diff
@@ -163,6 +163,7 @@ impl Project {
     // Check mandatory fields for crates.io
     pub fn check_mandatory_fields(&self) -> anyhow::Result<()> {
         let mut missing_fields = Vec::new();
+        let mut missing_versions = Vec::new();
```

Comment:
```suggestion
        let mut missing_versions_errors = Vec::new();
```

Reply:
Done. I lost an 's' and used `missing_version_errors` instead of `missing_versions_errors`